### PR TITLE
Add AXL to prices_tokens.sql

### PR DESCRIPTION
**Pull request by fahimahmedx**
Brief comments on the purpose of your changes:
These tokens are the lowercase contract addresses taken from: https://docs.axelar.dev/resources/mainnet#assets
The addresses exist on etherscan its variants for other chains.
A concern is that AXL is not active on coinpaprika. Could the address still be added, and if not how would AXL be set to active on coinpaprika?

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -55,6 +55,10 @@ VALUES
     ("ach-alchemy","arbitrum","ALCH","0x0e15258734300290a651fdbae8deb039a8e7a2fa",18),
     ("apex-apexit-finance","arbitrum","APEX","0x61a1ff55c5216b636a294a07d77c6f4df10d3b56",18),
     ("ata-automata","arbitrum","ATA","0xac9ac2c17cdfed4abc80a53c5553388575714d03",18),
+    ("axl-axelar","arbitrum","AXL","0x23ee2343b892b1bb63503a4fabc840e0e2c6810f",6),
+    ("axlusdc-axelar-wrapped-usdc","arbitrum","axlUSDC","0xeb466342c4d449bc9f53a865d5cb90586f405215",6),
+    ("axlusdt-axelar-wrapped-usdt","arbitrum","axlUSDT","0x7f5373ae26c3e8ffc4c77b7255df7ec1a9af52a6",6),
+    ("axlatom-axelar-wrapped-atom","arbitrum","axlATOM","0x06bee9e7238a331b68d83df3b5b9b16d5dba83ff",6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:61]:
```suggestion
    ("axlusdc-axelar-usd-coin","arbitrum","axlUSDC","0xeb466342c4d449bc9f53a865d5cb90586f405215",6),
    ("axlusdt-axelar-usd-tether","arbitrum","axlUSDT","0x7f5373ae26c3e8ffc4c77b7255df7ec1a9af52a6",6),
    ("axlatom-axelar-wrapped-atom","arbitrum","axlATOM","0x06bee9e7238a331b68d83df3b5b9b16d5dba83ff",6),
```

**Review by Hosuke — CHANGES_REQUESTED** [review]
Hi @fahimahmedx , sorry we could not add `axl-axelar` as this token-id is not available in coinpaprika api.

( Moreover `AXL` may have token-id `axl-axelar-network` but it is not active as well. 
API: https://api.coinpaprika.com/v1/coins/axl-axelar-network )

Please fix the usdc and usdt token-id as well.
axlUSDC API: https://api.coinpaprika.com/v1/coins/axlusdc-axelar-usd-coin
axlUSDT API: https://api.coinpaprika.com/v1/coins/axlusdt-axelar-usd-tether
axlATOM API: https://api.coinpaprika.com/v1/coins/axlatom-axelar-wrapped-atom

File: models/prices/prices_tokens.sql [hunk 2]
```diff
@@ -55,6 +55,9 @@ VALUES
     ("ach-alchemy","arbitrum","ALCH","0x0e15258734300290a651fdbae8deb039a8e7a2fa",18),
     ("apex-apexit-finance","arbitrum","APEX","0x61a1ff55c5216b636a294a07d77c6f4df10d3b56",18),
     ("ata-automata","arbitrum","ATA","0xac9ac2c17cdfed4abc80a53c5553388575714d03",18),
+    ("axlusdc-axelar-usd-coin","arbitrum","axlUSDC","0xeb466342c4d449bc9f53a865d5cb90586f405215",6),
+    ("axlusdt-axelar-usd-tether","arbitrum","axlUSDT","0x7f5373ae26c3e8ffc4c77b7255df7ec1a9af52a6",6),
+    ("axlatom-axelar-wrapped-atom","arbitrum","axlATOM","0x06bee9e7238a331b68d83df3b5b9b16d5dba83ff",6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:60]: Contract confirmation:
axlUSDC: https://arbiscan.io/token/0xeb466342c4d449bc9f53a865d5cb90586f405215
axlUSDT: https://arbiscan.io/token/0x7f5373ae26c3e8ffc4c77b7255df7ec1a9af52a6
axlATOM: https://arbiscan.io/token/0x06bee9e7238a331b68d83df3b5b9b16d5dba83ff

File: models/prices/prices_tokens.sql [hunk 3]
```diff
@@ -128,6 +131,9 @@ VALUES
     ("zz-zigzag","arbitrum","ZZ","0xada42bb73b42e0472a994218fb3799dfcda21237",18),
 
     ("aave-new","avalanche_c","AAVE.e","0x63a72806098bd3d9520cc43356dd78afe5d386d9",18),
+    ("axlusdc-axelar-usd-coin", "avalanche_c", "axlUSDC", "0xfab550568c688d5d8a52c7d794cb93edc26ec0ec", 6),
+    ("axlusdt-axelar-usd-tether", "avalanche_c", "axlUSDT", "0xf976ba91b6bb3468c91e4f02e68b37bc64a57e66", 6),
+    ("axlatom-axelar-wrapped-atom", "avalanche_c", "axlATOM", "0x80d18b1c9ab0c9b5d6a6d5173575417457d00a12", 6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:136]: Contract confirmation:
axlUSDC: https://snowtrace.io/token/0xfab550568c688d5d8a52c7d794cb93edc26ec0ec
axlUSDT: https://snowtrace.io/token/0xf976ba91b6bb3468c91e4f02e68b37bc64a57e66
axlATOM: https://snowtrace.io/token/0x80d18b1c9ab0c9b5d6a6d5173575417457d00a12

File: models/prices/prices_tokens.sql [hunk 4]
```diff
@@ -166,6 +172,7 @@ VALUES
     ("ant-aragon", "bnb", "ANT", "0x43f3918ff115081cfbfb256a5bde1e8d181f2907", 18),
     ("atom-cosmos", "bnb", "ATOM", "0x0eb3a705fc54725037cc9e008bdede697f62f335", 18),
     ("auto-auto", "bnb", "AUTO", "0x1cff458b364d0d328d4c9d59d10be7d22d01953d", 18),
+    ("axlusdc-axelar-usd-coin", "bnb", "axlUSDC", "0x4268b8f0b87b6eae5d897996e6b845ddbd99adf3", 6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:175]: Contract confirmation:
axlUSDC: https://bscscan.com/token/0x4268b8f0b87b6eae5d897996e6b845ddbd99adf3

File: models/prices/prices_tokens.sql [hunk 5]
```diff
@@ -354,6 +361,7 @@ VALUES
     ("aud-australian-dollar-token", "ethereum", "ibAUD", "0xfafdf0c4c1cb09d430bf88c75d88bb46dae09967", 18),
     ("audio-audius", "ethereum", "AUDIO", "0x18aaa7115705e8be94bffebde57af9bfc265b998", 18),
     ("awc-atomic-wallet-coin", "ethereum", "AWC", "0xad22f63404f7305e4713ccbd4f296f34770513f4", 8),
+    ("axlatom-axelar-wrapped-atom", "ethereum", "axlATOM", "0x27292cf0016e5df1d8b37306b2a98588acbd6fca", 6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:364]: Contract confirmation:
axlATOM: https://etherscan.io/token/0x27292cf0016e5df1d8b37306b2a98588acbd6fca

File: models/prices/prices_tokens.sql [hunk 6]
```diff
@@ -1738,6 +1746,9 @@ VALUES
 
     ("aave-new","polygon","AAVE","0xd6df932a45c0f255f85145f286ea0b292b21c90b",18),
     ("ageur-ageur","polygon",'agEUR',"0xe0b52e49357fd4daf2c15e02058dce6bc0057db4",18),
+    ("axlusdc-axelar-usd-coin", "polygon", "axlUSDC", "0x750e4c4984a9e0f12978ea6742bc1c5d248f40ed", 6),
+    ("axlusdt-axelar-usd-tether", "polygon", "axlUSDT", "0xceed2671d8634e3ee65000edbbee66139b132fbf", 6),
+    ("axlatom-axelar-wrapped-atom", "polygon", "axlATOM", "0x33f8a5029264bcfb66e39157af3fea3e2a8a5067", 6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1751]: Contract confirmation:
axlUSDC: https://polygonscan.com/token/0x750e4c4984a9e0f12978ea6742bc1c5d248f40ed
axlUSDT: https://polygonscan.com/token/0xceed2671d8634e3ee65000edbbee66139b132fbf
axlATOM: https://polygonscan.com/token/0x33f8a5029264bcfb66e39157af3fea3e2a8a5067

File: models/prices/prices_tokens.sql [hunk 7]
```diff
@@ -1790,6 +1801,9 @@ VALUES
     ("ftm-fantom", "fantom", "WFTM", "0x21be370D5312f44cB42ce377BC9b8a0cEF1A4C83", 18),
     ("aave-new", "fantom", "AAVE", "0x6a07A792ab2965C72a5B8088d3a069A7aC3a993B", 18), 
     ("avax-avalanche", "fantom", "AVAX", "0x511D35c52a3C244E7b8bd92c0C297755FbD89212", 18), 
+    ("axlusdc-axelar-usd-coin", "fantom", "axlUSDC", "0x1b6382dbdea11d97f24495c9a90b7c88469134a4", 6),
+    ("axlusdt-axelar-usd-tether", "fantom", "axlUSDT", "0xd226392c23fb3476274ed6759d4a478db3197d82", 6),
+    ("axlatom-axelar-wrapped-atom", "fantom", "axlATOM", "0x3bb68cb55fc9c22511467c18e42d14e8c959c4da", 6),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1806]: Contract confirmation:
axlUSDC: https://ftmscan.com/token/0x1b6382dbdea11d97f24495c9a90b7c88469134a4
axlUSDT: https://ftmscan.com/token/0xd226392c23fb3476274ed6759d4a478db3197d82
axlATOM: https://ftmscan.com/token/0x3bb68cb55fc9c22511467c18e42d14e8c959c4da

**Review by Hosuke — APPROVED** [review]
LGTM.✅
- `token_id` matches coinpaprika id ✅
- `is_active: true` ✅
- contract addresses are lowercase and correct ✅

Thank you @fahimahmedx

**Review by dot2dotseurat — APPROVED** [review]

**Pull request merged**